api/CredentialsContainer.json

```diff
@@ -109,6 +109,218 @@
               "deprecated": false
             }
           },
+          "attestation": {
+            "__compat": {
+              "description": "<code>attestation</code> option",
+              "spec_url": "https://w3c.github.io/webauthn/#dom-publickeycredentialcreationoptions-attestation",
+              "support": {
+                "chrome": {
+                  "version_added": "67"
+                },
+                "chrome_android": {
+                  "version_added": "70"
+                },
+                "edge": {
+                  "version_added": "18"
+                },
+                "firefox": {
+                  "version_added": "60"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "13"
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": {
+                  "version_added": false
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": true,
+                "deprecated": false
+              }
+            },
+            "direct": {
+              "__compat": {
+                "description": "<code>direct</code> attestation conveyance preference",
```

Comment:
I think we can omit the description (see [`text-align: center`](https://github.com/mdn/browser-compat-data/blob/9a1ea2af49099efcce1f6eea48cc2c5b38753365/css/properties/text-align.json#L47-L50)):

```suggestion
```